airflow/www/templates/airflow/dags.html

```diff
@@ -191,11 +191,11 @@ <h2>DAGs</h2>
                 </a>
 
                 <!-- Delete -->
-                <a href="{{ url_for('airflow.delete', dag_id=dag.dag_id) }}"
-                  onclick="return confirmDeleteDag('{{ dag.safe_dag_id }}')">
+                <!-- Use dag_id instead of dag.dag_id, because the DAG might not exist in the webserver's DagBag -->
```

Comment:
Minor nit: this includes the comment in thee HTML which we don't need. If you think the comment is useful then:

```suggestion
                {# Use dag_id instead of dag.dag_id, because the DAG might not exist in the webserver's DagBag #}
```